<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>outputDialog</class>
 <widget class="QDialog" name="outputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>OLAgenLogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select Target of Interest</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="targetLst">
      <property name="cursor" stdset="0">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="csvExportBtn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Export to .csv</string>
   </property>
  </widget>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>550</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Return Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="plsUploadLbl">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>580</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>721</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="ligTab">
    <attribute name="title">
     <string>Ligation Probes</string>
    </attribute>
    <widget class="QTableWidget" name="probeTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>671</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>SNP</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Variable Probe (5' to 3')</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Variable WT Probe (5' to 3')</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Common Probe (5' to 3')</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addSOIBtn">
     <property name="geometry">
      <rect>
       <x>502</x>
       <y>240</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add Set of Interest</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="soiTab">
    <attribute name="title">
     <string>Sets of Interest</string>
    </attribute>
    <widget class="QTableWidget" name="soiTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>671</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Set ID</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Target</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SNP</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Primers Generated?</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="genPrimerBtn">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>240</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Primers</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeSetBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="primerTab">
    <attribute name="title">
     <string>Primer Candidates</string>
    </attribute>
    <widget class="QTableWidget" name="primerTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>671</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Set ID</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Forward Primer</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reverse Primer</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hydrolysis Probe</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Primer Type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Citation</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="removePrimerBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>240</y>
       <width>151</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Candidate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genFullReagent">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>240</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Full Reagents</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="allTab">
    <attribute name="title">
     <string>All Reagents</string>
    </attribute>
    <widget class="QTableWidget" name="reagentTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>671</width>
       <height>231</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Set ID</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Target</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SNP</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VP+Fwd+Hydr</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VP WT+Fwd</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CP+Rev</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Forward Primer</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reverse Primer</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hydrolysis Probe</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Primer Type</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Citation</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
